locations/location_mgr.go

```diff
@@ -170,9 +170,9 @@ func (mgr *locationManager) GetLocations() ([]LocationConfiguration, error) {
 }
 
 // GetLocationProperties returns properties configured for a given location
-func (mgr *locationManager) GetLocationProperties(locationName, locationType string) (config.DynamicMap, error) {
```

Comment:
I think it's quite confusing to rename locationType into infrastructureType ?

Reply:
I looked to GetLocationPropertiesForNode usage and noted that its always called using an infrastructureType as the 4th parameter : GetLocationPropertiesForNode(ctx, deploymentID, nodeName, infrastructureType).
For example:
https://github.com/ystia/yorc/blob/bugfix/GH-550-check-loc-type/prov/terraform/google/generator.go#L55

Reply:
Terraform generators in particular use the infrastructure type found in the compute node.
It really correspond to a supported infrastructure type.

Reply:
The difficulty here is to cope with 2 _locationTypes_ : the argument of the function call and the type of the location identified by the locationName, also argument of the function call.
I made the choise to use 'locationType' for the second

Reply:
Yep, externally for client, it 's an infrastructure type. But internally to the locations API, it's what has been called location type

Reply:
In this case, maybe we should change infrastructureType to locationType in the functions calling GetLocationPropertiesForNode in order to be coherent ?

Reply:
Not sure it's really useful. WYT @loicalbertin @laurentganne  ?

Reply:
actually we re-used the terms used by A4C which introduced the notion of location.
In A4C, a location is defined by  a name and an infrastructure type:
![newLocation](https://user-images.githubusercontent.com/33217305/77637620-3bd66b80-6f56-11ea-9b71-d9d22ee965be.png)

Reply:
Ok

Reply:
I'm afraid we have now both infrastructureType and locationType in the locations package. This can be confusing